docs/authorization-configuration.md

```diff
@@ -134,28 +134,20 @@ auth: {
 - **type**: number
 - **description**: is the number of seconds before an access token expires and triggers silent token refresh.
 - **default**: `60` seconds
-#### thirdPartyCookiesScriptLocation
-- **type**: string
-- **description**: is the URL to the page containing third-party cookies support check. For details, see [Third-party cookies and silent token refresh section](#Third-party-cookies-and-silent-token-refresh).
 #### userInfoFn
 - **type**: function
 - **description**: provides a function to get user information. It returns a promise of a **userinfo** object which can contain **name**, **email** and **picture** (value is a URL to the image). **Name** or **email** are displayed in the profile drop-down menu and the user’s profile picture is displayed in the top navigation.
 #### profileStorageInterceptorFn
 - **type**: function
 - **description**: provides a function to mutate the profile values of the [JSON Web Token (JWT)](https://jwt.io) before it gets stored in Luigi. It allows the removal of values like **email** to comply with data privacy restrictions. Since it is async, you could use it to enrich the profile data, but it should not get mixed up with **userInfoFn** which is specifically designed to define user information.
 
-### Third-party cookies and silent token refresh
+### Silent token refresh
 
 The OpenID Connect configuration allows you to specify the **automaticSilentRenew** option. When set to `true`, Luigi attempts to automatically renew the token in the background before it expires. Be aware that this mechanism requires the browser to support [third-party cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).
```

Comment:
I went through this silent token part section once more. And I think the following line:

> The OpenID Connect configuration allows you to specify the **automaticSilentRenew** option. When set to `true`, Luigi attempts to automatically renew the token in the background before it expires. Be aware that this mechanism requires the browser to support [third-party cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).

The text indicates that **third party cookies**(TPC) and **silent renew** come together as a 'package'. So if we remove **TPC** check and all support for **TPC** it makes no sense to keep the **automaticSilentRenew** option altogether. 

We were able so far to offer this functionality ourselves by leveraging **TPC**. If **TPC** is gone then we cannot offer the functionality anymore. 

I think then the whole **### Silent token refresh** can go altogether, and with it the silent token refresh functionality. 

Looking back, this Warning on here, indicates that without TPC, silent renewal might not work. [Check Warning](https://github.com/SAP/luigi/pull/180/commits/79ecacebdbe16f84dc113a3516a68b7c6b8ec8e9#diff-373c23867778f0930e352885d2def2a1a72f90ad22320dc173e3c421c6dd621eR6)  on the [Pull Request that first added Token Refresh ](https://github.com/SAP/luigi/pull/180/files). 

I would guess in this case developers would have to implement that on their own. Will ask this again in the daily.